inc/metadata/namespace.php

```diff
@@ -260,63 +265,26 @@ function book_information_to_schema( $book_information, $network_excluded_direct
 		}
 	}
 
-	if ( isset( $book_information['pb_authors'] ) ) {
-		$authors = oxford_comma_explode( $book_information['pb_authors'] );
-		foreach ( $authors as $author ) {
-			$book_schema['author'][] = [
-				'@type' => 'Person',
-				'name' => $author,
-			];
-		}
-	}
-
-	if ( isset( $book_information['pb_editors'] ) ) {
-		$editors = oxford_comma_explode( $book_information['pb_editors'] );
-		foreach ( $editors as $editor ) {
-			$book_schema['editor'][] = [
-				'@type' => 'Person',
-				'name' => $editor,
-			];
-		}
-	}
-
-	if ( isset( $book_information['pb_translators'] ) ) {
-		$translators = oxford_comma_explode( $book_information['pb_translators'] );
-		foreach ( $translators as $translator ) {
-			$book_schema['translator'][] = [
-				'@type' => 'Person',
-				'name' => $translator,
-			];
-		}
-	}
-
-	if ( isset( $book_information['pb_reviewers'] ) ) {
-		$reviewers = oxford_comma_explode( $book_information['pb_reviewers'] );
-		foreach ( $reviewers as $reviewer ) {
-			$book_schema['reviewedBy'][] = [
-				'@type' => 'Person',
-				'name' => $reviewer,
-			];
-		}
-	}
-
-	if ( isset( $book_information['pb_illustrators'] ) ) {
-		$illustrators = oxford_comma_explode( $book_information['pb_illustrators'] );
-		foreach ( $illustrators as $illustrator ) {
-			$book_schema['illustrator'][] = [
-				'@type' => 'Person',
-				'name' => $illustrator,
-			];
-		}
-	}
-
-	if ( isset( $book_information['pb_contributors'] ) ) {
-		$contributing_authors = oxford_comma_explode( $book_information['pb_contributors'] );
-		foreach ( $contributing_authors as $contributor ) {
-			$book_schema['contributor'][] = [
-				'@type' => 'Person',
-				'name' => $contributor,
-			];
+	$contributors = new Contributors();
+	foreach ( $contributors->valid as $contributor_type ) {
```

Comment:
Instead multiple if for each type of contributors and repeat the logic, I used the valid public property